dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2888,3 +2888,108 @@ def test_parquet_pyarrow_write_empty_metadata_append(tmpdir):
         append=True,
         ignore_divisions=True,
     )
+
+
+def test_read_write_partitioned_overwrite_is_true(tmpdir):
```

Comment:
```suggestion
def test_read_write_overwrite_is_true(tmpdir, engine):
```
Lets use the `engine` fixture.

I don't have strong feelings about the name, but I do find it confusing to use the overloaded `partitioned` term here (since it is used ind `dask.dataframe` for both DataFrame-collection and parquet-file "partitions").